master/getting-started/openshift/installation.md

```diff
@@ -1,10 +1,28 @@
 ---
 title: Installing Calico on OpenShift
+redirect_from: latest/getting-started/openshift/installation
```

Comment:
This redirect line should only be present in the folder that corresponds to the latest release, i.e., 2.5 at present (not master).